docs/treemap.md

```diff
@@ -202,3 +208,61 @@ Pass in a function that will be used to sort the nodes, for more information see
 
 Scale properties for the `color` scale. If `color` property is not passed in the data object, each new section of the chart gets the next color (e. g. the `'category'` scale is applied).
 Please refer to [Scales and Data](scales-and-data.md) for more information about scales.
+
+#### getSize
```

Comment:
Maybe include you example in the docs via a <!-- INJECT:"SimpleTreemapWithLink" --> type thing